docker/docker-compose.yml

```diff
@@ -7,10 +7,10 @@ services:
     container_name: watchtower
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-    environment:
-      - WATCHTOWER_NOTIFICATIONS=slack
-      - WATCHTOWER_NOTIFICATION_SLACK_HOOK_URL=$SLACK_HOOK_URL
-      - WATCHTOWER_NOTIFICATION_SLACK_IDENTIFIER=Devnet-Watchtower-Service
+    # environment:
+    #   - WATCHTOWER_NOTIFICATIONS=slack
+    #   - WATCHTOWER_NOTIFICATION_SLACK_HOOK_URL=$SLACK_HOOK_URL
+    #   - WATCHTOWER_NOTIFICATION_SLACK_IDENTIFIER=Devnet-Watchtower-Service
```

Comment:
Commented code, just in case someone misses it. <_<"

Reply:
Removed.